src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -607,23 +668,18 @@ public async Task VerifyBankAsync(Guid organizationId, int amount1, int amount2)
     public async Task<Tuple<Organization, OrganizationUser>> SignUpAsync(OrganizationSignup signup,
         bool provider = false)
     {
-        var plan = StaticStore.PasswordManagerPlans.FirstOrDefault(p => p.Type == signup.Plan);
-        if (plan is not { LegacyYear: null })
-        {
-            throw new BadRequestException("Invalid plan selected.");
-        }
-
-        if (plan.Disabled)
+        var plans = StaticStore.Plans.Where(p => p.Type == signup.Plan).ToList();
```

Comment:
Similar to above - narrow this down to only the plans you need immediately.

Reply:
resolved